src/ducks/workflow.js

```diff
@@ -98,9 +102,40 @@ const toggleSelection = (show) => {
   };
 };
 
+const fetchPhaseTwoWorkflows = () => {
+  return (dispatch) => {
+    const c = config;
+    const allWorkflows = {
```

Comment:
This being an object doesn't really seem useful to you since in the component you convert it back to an array anyway. I would start with an array of workflow ids to begin with to pass into the request directly:
```
apiClient.type('workflows').get(arrayOfWorkflowIds).then(allWorkflows => dispatch({ type: FETCH_ALL_WORKFLOWS, allWorkflows})).catch(err => console.error(err))
```

Then in your component, you can iterate over an array directly since the response will be an array of workflows.

Reply:
I want to keep `allWorkflows` as an object since the user could be swapping between workflows throughout the project. Storing all workflows as an object seemed an easier way to switch to a different workflow if the object used workflow ids as object keys.